<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PreguntaForm</class>
 <widget class="QWidget" name="PreguntaForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Tema:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblTema">
     <property name="text">
      <string>Desconocido</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Concepto:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QTextEdit" name="txtConcepto">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>TÃ©rmino:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="cmbTerminos"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Pregunta:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="lblContador">
     <property name="text">
      <string>0/0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnEnd">
     <property name="text">
      <string>Finalizar</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Recursos/comprobado.png</normaloff>Recursos/comprobado.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnNext">
     <property name="text">
      <string>   Siguiente</string>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>Recursos/intercambio.png</normaloff>Recursos/intercambio.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
